### PR TITLE
Added available regions as at April 2022

### LINKED ISSUE #519: Include available cloud region for all 3 providers
As at April 2022

### PR DESCRIPTION
**Pull request by ylorph**
Removed uncessary comments about Azure regions availability.
Closes #519

### CODE REVIEW

File: docs/cloud/provision/README.md [hunk 1]
```diff
@@ -567,3 +622,12 @@ Due to the burstable CPU class, CPU shares are limited, this results in the foll
 | **M32**  | Production                     | 12M                   | 1 TB            | 250                      |
 | **M64**  | Production                     | 30M                   | 2 TB            | 500                      |
 | **M128** | Production                     | 62M                   | 4 TB            | 500                      |
+
+## Available Regions 
+
+This is the list of availble regions as at April 2022
```
Comment (oskardudycz) [inline docs/cloud/provision/README.md]: **CONCERN:** Something looks missing here

File: docs/cloud/provision/README.md [hunk 2]
```diff
@@ -336,31 +385,6 @@ We're aware that Azure Premium SSD volumes have a strict IOPS limit and this lim
 
 Azure resource provisioning speed will almost always be slower and more highly variable than other providers. Please account for this when interacting with provisioned resources on Azure.
 
-#### Available regions
-
-At the moment, you can deploy EventStoreDB clusters only in regions with three or more availability zones, and many Azure regions do not have the minimum number of availability zones to support the deployment of managed EventStoreDB clusters. In the future, we plan to allow the deployment of single node topologies to all regions. We'll also introduce a new topology that allows cluster deployments to a single availability zone.
```
Comment (oskardudycz) [inline docs/cloud/provision/README.md:341]: **QUESTION:** Why was this explanation removed?

File: docs/cloud/provision/README.md (hunk 1 above)
Comment (ylorph) [reply docs/cloud/provision/README.md]: 😅 moved it , forgot to remove it completely

Comment (ylorph) [reply docs/cloud/provision/README.md]: Corrected @oskardudycz

File: docs/cloud/provision/README.md (hunk 2 above)
Comment (ylorph) [reply docs/cloud/provision/README.md:341]: Gonna move that to the FAQ section , since it's also true for _any_ provider.

**Review by alexeyzimarev — APPROVED** [review]

**Pull request merged**